app/components/cards/PostFull.jsx

```diff
@@ -134,6 +134,20 @@ class PostFull extends React.Component {
             this.state !== nextState
     }
 
+    componentDidMount() {
+        const headerPic = document.querySelector('.PostFull__header .Userpic');
+        if (headerPic) headerPic.addEventListener("click", this.showAuthorPreview, false);
+    }
```

Comment:
The react's way to do it is to wrap Userpick with <a> and use onClick event:
<a href="#" onClick={}><Userpic/></a>